sympy/utilities/lambdify.py

```diff
@@ -450,6 +455,48 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def should_dummify(expr, printer=None):
```

Comment:
I initially used sstr for string conversion purposes (which is used in keyword detection & regex matching) but after the suggestion of @asmeurer , I changed to use the printer which is used by lambdify function . Thus, I would be needing the printer as an argument.

This function returns the boolean which is the value of dummify to be used